tests/sdk/test_session_context.py

```
@@ -0,0 +1,625 @@
+    try:
+        import sdks.python.src.honcho.session_context as session_context_module
+
+        # Verify the module loaded successfully
+        assert hasattr(session_context_module, "SessionContext")
+
+        # Verify the type annotations that depend on line 9 work
+        session_context = session_context_module.SessionContext
+        assert hasattr(session_context, "to_openai")
+        assert hasattr(session_context, "to_anthropic")
+
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)